unit-tests/src/test/scala/scala/scalanative/unsafe/CStringSuite.scala

```diff
@@ -11,29 +10,52 @@ object CStringSuite extends tests.Suite {
     fromCString(c"")
     fromCString(c"no escapes")
     fromCString(c"\'"); fromCString(c"\\'")
-    fromCString(c"\?"); fromCString(c"\\?")
+    fromCString(c"\\?")
     fromCString(c"\\")
-    fromCString(c"\a"); fromCString(c"\\a")
-    fromCString(c"\b"); fromCString(c"\\b")
-    fromCString(c"\f"); fromCString(c"\\f")
-    fromCString(c"\n"); fromCString(c"\\n")
-    fromCString(c"\r"); fromCString(c"\\r")
-    fromCString(c"\t"); fromCString(c"\\t")
-    fromCString(c"\v"); fromCString(c"\\v")
-    fromCString(c"\012\x6a")
-    fromCString(c"%s \\t %.2f %s/s\00")
+    fromCString(c"\\a")
+    fromCString(c"\\v")
+    fromCString(c"%s \\t %.2f %s/s\x00")
 
     // uncomment the following to trigger compilation errors for testing
     // fromCString(c"\")     // error at NIR
     // fromCString(c"\x2ae") // error at NIR
     // fromCString(c"\"")    // error at Scala compiler
+    // fromCString(c"\?")
+    // fromCString(c"\v")
+    // fromCString(c"\a")
+    // fromCString(c"\012")
   }
 
   test("""the value of c"..." literals""") {
+
+    assertEquals("\b", fromCString(c"\b"))
+    assertEquals("\\b", fromCString(c"\\b"))
+
+    assertEquals("\f", fromCString(c"\f"))
+    assertEquals("\\f", fromCString(c"\\f"))
+
     assertEquals("\t", fromCString(c"\t"))
     assertEquals("\\t", fromCString(c"\\t"))
-    assertEquals("\u0020\u0020\u0061\u0062", fromCString(c"\040\40\141\x62"))
-    assertEquals("\\\\v\t\\abc", fromCString(c"\\\v\t\\abc"))
+
+    assertEquals("\\n", fromCString(c"\\n"))
+    assertEquals("\n", fromCString(c"\n"))
+
+    assertEquals("\\r", fromCString(c"\\r"))
+    assertEquals("\r", fromCString(c"\r"))
+
+    assertEquals("""
```

Comment:
@lolgab will be happy that you fixed this. Me too!  

What do you do with your pile of dead bugs?

Reply:
Thanks for the mention! 😃
Yes, I'm happy, especially now that I don't have to fix my PR because this one supersedes mine 😄 ( didn't have too much time and willpower lately to finish it :( ).